ElectronClient/app/gui/SideBar.jsx

```diff
@@ -440,10 +446,40 @@ class SideBarComponent extends React.Component {
 	makeHeader(key, label, iconName, extraProps = {}) {
 		const style = this.style().header;
 		const icon = <i style={{ fontSize: style.fontSize * 1.2, marginRight: 5 }} className={"fa " + iconName} />;
+
+		if (extraProps.toggleblock || extraProps.onClick) {
```

Comment:
Please use camelCase (i.e. should be "toggleBlock")

Reply:
Hey @laurent22 - This is a custom DOM attribute. I think React does not allow us to camelCase these.

Reply:
Hi @Abijeet, you're correct! Let's keep that way then.